src/h5web/toolbar/controls/DomainSlider/ColorBarGradient.tsx

```diff
@@ -0,0 +1,38 @@
+import { range } from 'lodash';
+import type { SVGProps } from 'react';
+import type { ColorMap } from '../../../../packages/lib';
+import { getInterpolator } from '../../../vis-packs/core/heatmap/utils';
+
+const GRADIENT_PRECISION = 1 / 20;
+const GRADIENT_RANGE = range(0, 1 + GRADIENT_PRECISION, GRADIENT_PRECISION);
```

Comment:
I could not find an easy way to reuse the `getColorStops` and `getLinearGradient` methods as the `svg` gradient is quite different from the CSS implementation.